log/common.go

```diff
@@ -0,0 +1,36 @@
+package log
+
+import (
+	"io"
+	"os"
+)
+
+// For debugging purposes we can change the output-writer
+var stdOut io.Writer
+var stdErr io.Writer
+
+func init() {
+	stdOut = os.Stdout
+	stdErr = os.Stderr
+}
+
+const (
+	lvlWarning = iota - 20
+	lvlError
+	lvlFatal
+	lvlPanic
+	lvlInfo
+	lvlPrint
+)
+
+// Format indicates how the prints will be shown
+var Format = FormatLvl
```

Comment:
I see. Why I wanted to have in one central place is because they do the same thing and it *can* be used in both situations:
* In cothorityd, you want to have debug output AND general purpose output.
* In client apps, you also want the same as you would want to debug also (connection opening, closing, error while verifying signature etc).

In both cases, only the *format* changes.
The way I saw the merging between `ui` and `dbg` was more stg like there is **only** `LvlXXX` or `Info,Warn,Debug` but not both, because like you said, it's inconsistent.
So a client would set the right format `FormatPython` and cothorityd would set `FormatDbg` and both will uses the same *unified* methods (`Info`, `Warn`, `Debug`... I prefer these names because it's clear when to use them unlike `LvlXX` where we don't even use half of it)

Lol we have an issue here, so it may be a good time to discuss with @Liamsi on the whiteboard in some point soon.